assets/component-localization-form.css

```diff
@@ -238,31 +238,31 @@ noscript .localization-selector.link {
   min-height: 4.4rem;
 }
 
-.js .header__icons--localization .localization-form__select {
+.js .header__localization .localization-form__select {
   padding: 0 2.7rem 0 1.2rem;
   width: max-content;
 }
 
-.header__icons--localization .localization-form:only-child .localization-form__select {
+.header__localization .localization-form:only-child .localization-form__select {
   margin: 0;
 }
 
-.header__icons--localization .localization-form__select > span {
+.header__localization .localization-form__select > span {
   max-width: 20ch;
   white-space: nowrap;
   text-overflow: ellipsis;
   overflow: hidden;
 }
 
-.header__icons--localization localization-form:only-child .localization-form__select > span {
+.header__localization localization-form:only-child .localization-form__select > span {
   max-width: 26ch;
 }
 
-.header__icons--localization .localization-form__select .icon-caret {
+.header__localization .localization-form__select .icon-caret {
   right: 0.8rem;
 }
 
-.header__icons--localization .disclosure__list-wrapper {
+.header__localization .disclosure__list-wrapper {
```

Comment:
There is one more `header__icons--localization` in this file that should be changed I think 🤔 
Maybe I misremember my recommendation 😅 

But we shouldn't have `header__icons--localization` in the `announcement-bar.liquid` file. 

Could we use `header__localization` everywhere instead ?

Reply:
I removed header__icons--localization and only kept header__localization in both the announcement bar and header :)